sdk/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpansProcessor.java

```diff
@@ -237,6 +237,7 @@ public BatchSpansProcessor build() {
 
   private static Thread newThread(Runnable runnable) {
     Thread thread = MoreExecutors.platformThreadFactory().newThread(runnable);
+    thread.setDaemon(true);
```

Comment:
discussion on the very very similar #1094 says this needs to go inside the try block because security managers can make this illegal.